config/database.rb

```diff
@@ -2,34 +2,52 @@
 
 def self.connect_to_database_prod
   begin
-    ActiveRecord::Base.establish_connection(
+    config = {
       :adapter  => "mysql2",
       :host     => "#{ENV["DATABASE_SERVICE_HOST"]}",
       :port     => "#{ENV["DATABASE_SERVICE_PORT"]}",
-      :database => "#{ENV["MYSQL_DATABASE"]}",
-      :password => "#{ENV["MYSQL_ROOT_PASSWORD"]}"
-    )
+      :database => "#{ENV["MYSQL_DATABASE"]}"
+    }
+    if ENV.key?("MYSQL_ROOT_PASSWORD")
+      config['password'] = "#{ENV["MYSQL_ROOT_PASSWORD"]}"
+    else
+      config['username'] = "#{ENV["MYSQL_USER"]}"
+      config['password'] = "#{ENV["MYSQL_PASSWORD"]}"
+    end
+
+    puts "Connecting to production database (#{config['username']}@#{config['host']}:#{config['port']})...\n"
```

Comment:
Are you sure? I moved this here from app.rb and \n was already there. I didn't see any empty lines in the logs.